pootle/apps/pootle_fs/migrations/0002_convert_localfs.py

```diff
@@ -106,30 +112,52 @@ def convert_to_localfs(apps, schema_editor):
     old_translation_path = settings.POOTLE_TRANSLATION_DIRECTORY
 
     for project in Project.objects.exclude(treestyle="pootle_fs"):
+        logger.debug("Converting project '%s' to pootle fs", project.code)
         proj_trans_path = str(PosixPath().joinpath(old_translation_path, project.code))
         proj_stores = Store.objects.filter(
-            translation_project__project=project).exclude(file="")
+            translation_project__project=project).exclude(file="").exclude(obsolete=True)
         _set_project_config(Config, project_ct, project)
         project.treestyle = "pootle_fs"
         project.save()
+
+        if project.disabled:
+            continue
+        if not os.path.exists(proj_trans_path):
+            logger.warn(
+                "Missing project ('%s') translation directory '%s', skipping",
```

Comment:
I would be more clear on what is being skipped. Someone might think the project is skipped while it was changed a few lines above.